src/reference/config/guards/azure-ad.md

```diff
@@ -0,0 +1,79 @@
+---
+redirectFrom: /reference/config/guards/guard-azure-ad.html
+shortTitle: azure-ad
+category:
+  - Guard
+tag:
+  - azure-ad
+---
+
+# azure-ad Guard
+
+Defines a guard with `Azure AD` support.
+
+The `azure-ad` guard uses public keys to verify the integrity of `access tokens` when identifying authorized subjects and their associated roles scope. The token issuer and audience can also be constrained to prevent access tokens from other applications from being reused inappropriately.
+
+Each verified access token has an expiration time, and an optional challenge window prior to the expiration time that can be used by specific protocol bindings to send a challenge to renew the access token before it expires.
+
+For details on public key discovery and rotation, see [Key Discovery & Rotation](../../../concepts/security/guard/azure-ad/README.md#key-discovery-rotation)
+
+```yaml {2}
+guards:
+  my_azure_ad_guard:
+    type: azure-ad
+    options:
+      audience: 00000000-1111-2222-3333-444444444444
```

Comment:
Is this a typical value for `audience`?
Perhaps something that represents the target API where the token is used to authorize?